src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/PropertyExpression.cs

```diff
@@ -38,55 +38,55 @@ private void OnPropertyChanged()
 
             if (Property.Type.IsEnumEx())
             {
-                CreateDataTypeExpressionIfNotCompatible(() => new StringExpression());
+                DataTypeExpression = new StringExpression();
```

Comment:
OK, do not create example app, I will just try to add checking for nullable too.